### PR TITLE
feature/AT-SCE-UI_Templates_AdminModule

### PR DESCRIPTION
**Pull request by MiltonNAR**
Handlebar views have been added to the admin module.

### CODE REVIEW

File: src/views/admin/index.hbs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div class="jumbotron mt-1">
+     <div class="container" style="text-align: left;">
+
+          <h1>Admin Module</h1>
+          <!--linea -->
+          <hr color="blue" size=3>
+           <h4>AT-SCE</h4>
+
+           <button type="button" class="btn btn-primary"> User</button>
```
Comment (at-internship) [inline src/views/admin/index.hbs]: `<a class="btn btn-primary" href="/admin/user">&raquo; Users</a>`

**Review by at-internship — CHANGES_REQUESTED** [review]

File: src/views/admin/index.hbs (hunk 1 above)
Comment (at-internship) [reply src/views/admin/index.hbs]: you can follow this view as reference https://github.com/at-internship/at-sso-ui/blob/develop/src/views/admin/index.hbs

**Comment by at-internship** [comment]
Approved

**Review by at-internship — APPROVED** [review]
Approved

**Review by irvingesp2190 — APPROVED** [review]

**Review by luishcastroc — APPROVED** [review]

**Pull request merged**